ui.config/src/main/content/jcr_root/apps/wknd/osgiconfig/config.author/com.adobe.granite.cors.impl.CORSPolicyImpl~wknd-graphql.cfg.json

```diff
@@ -0,0 +1,30 @@
+// Configuration created by Apache Sling JCR Installer
```

Comment:
at a glance, this appears to be the same for `config.author` and `config.publish` .. any reason not to consolidate down to `config`?

Reply:
I assume most implementations would have separate configs for author/publish. Since you would have a preview app connecting to the author vs a production app connecting to publish. However since this is just a sample maybe this makes it more confusing?